.gitignore

```diff
@@ -2,6 +2,7 @@
 *~
 *.aar
 *.apk
+*.exe
```

Comment:
Why is this needed?

Reply:
I'm developing on windows, when I run "go build" from "ebiten\cmd\ebitenmobile" directory it produces an exe. I felt we wouldn't want to checkin built binaries to ebiten repo (if we later want to, we could change this). I can remove it if you think this is not appropriate.

Reply:
This seems a different issue from what we are focusing on now. Let's remove this.